backend/README.md

```
@@ -170,3 +170,43 @@
+- Activate it:
+```shell
+source .venv/bin/activate
+```
+- Reinstall dependencies:
+```shell
+uv sync
+```
+
+### Run everything in docker: infrastructure + backend + celery
+
+- In root folder:
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)